src/encode.cpp

```diff
@@ -1034,52 +1035,24 @@ void encodeOp(State &st, mlir::tosa::DepthwiseConv2DOp op, bool) {
 
```

Comment:
Could you make a separate PR that only updates `encodeOp(State &st, mlir::tosa::DepthwiseConv2DOp ...)` please?
I will merge the PR first once it is ready.

Reply:
Let's make a PR that update encoding `mlir::tosa::DepthwiseConv2DOp` to use `getPaddedTensor2D` first.